react/features/aeternity/components/TipButton.js

```diff
@@ -255,12 +260,22 @@ class TipButton extends Component<Props, State> {
      * @returns {void}
      */
     async _onSendTip() {
-        if (!this.props.account) {
+        const { account } = this.props;
+
+        if (!account) {
+            return;
+        }
```

Comment:
and this we don't need until we will not show the button is there is no valid prop